plugins/vestige/vestige.cpp

```diff
@@ -127,6 +177,13 @@ void vestigeInstrument::loadSettings( const QDomElement & _this )
 	{
 		m_plugin->loadSettings( _this );
 
+		m_plugin->createUI( nullptr );
```

Comment:
Why do we need this call? At this moment `loadFile` already have created UI, so this call will cause the bug reported by @qnebra.

Reply:
Hmmm, interesing.

Reply:
@PhysSong this call make a issue. I commented this out, compile lmms and check. Cloning works correctly.